src/masoniteorm/commands/MigrateRefreshCommand.py

```diff
@@ -9,12 +9,22 @@ class MigrateRefreshCommand(Command):
     migrate:refresh
         {--c|connection=default : The connection you want to run migrations on}
         {--d|directory=databases/migrations : The location of the migration directory}
+        {--s|seed=? : Seed database after refresh. The seeder to be ran can be provided in argument}
```

Comment:
Means an optional option

https://cleo.readthedocs.io/en/latest/introduction.html#using-options

Using just a option isn't possible If the absent If It it's because isn't define or define with no value

Reply:
ah that's nice I always wondered how to have an optional arg without value but which can take value if needed 💪